.github/workflows/daily-php.yml

```diff
@@ -0,0 +1,36 @@
+name: 'Daily PHP Tests'
+
+on:
+  pull_request:
+  schedule:
+    - cron:  '0 0 * * *'
+
+jobs:
+    daily-test-php:
+        name: "Test PHP"
+        runs-on: ubuntu-18.04
+        strategy:
+            fail-fast: false
+            matrix:
+                wordpress: ['latest', 'nightly']
+                woocommerce: ['latest', 'nightly']
+                exclude:
+                    - {'wordpress': 'nightly', 'woocommerce': 'nightly'}
+        steps:
+            - name: Check out repository code
+              uses: actions/checkout@v2
+            - name: Setup Node.js
+              uses: actions/setup-node@v2-beta
+              with:
+                  node-version: '14'
+            - name: Build
+              run: |
+                  npm run build:feature-config	
+                  composer install  --no-dev
+              shell: bash
+            - name: Run the PHP unit tests
+              env:
+                  WP_VERSION: ${{ matrix.woocommerce }}
```

Comment:
This should be `matrix.wordpress`.

Reply:
Ah, nice catch! Thank you it's now fixed in e987e13